mkt/purchase/webpay.py

```diff
@@ -206,6 +228,20 @@ def simulated_postback(contrib, trans_id):
     return http.HttpResponse(trans_id)
 
 
+def free_postback(request, contrib, trans_id, user_profile):
+    log.info('Got free product postback: fulfilling pruchase for contrib={c}; '
+             'trans={t}; user={u}'.format(c=contrib,
+                                          t=trans_id, u=user_profile))
```

Comment:
Since you have this log message here, you can probably remove the same one from above: https://github.com/mozilla/zamboni/pull/2970/files#diff-bff80a5504e028c0f04952576a512f85R175